libsolidity/analysis/GlobalContext.cpp

```diff
@@ -51,15 +51,15 @@ m_magicVariables(vector<shared_ptr<MagicVariableDeclaration const>>{make_shared<
 					make_shared<MagicVariableDeclaration>("keccak256",
 							make_shared<FunctionType>(strings(), strings{"bytes32"}, FunctionType::Kind::SHA3, true, StateMutability::Pure)),
 					make_shared<MagicVariableDeclaration>("log0",
-							make_shared<FunctionType>(strings{"bytes32"}, strings{}, FunctionType::Kind::Log0)),
+							make_shared<FunctionType>(strings{"bytes32"}, strings{}, FunctionType::Kind::Log0, false, StateMutability::NonPayable)),
```

Comment:
`NonPayable` is the default setting

Reply:
ok, now I'm feeling stupid :-)

Reply:
Ah and now I also remember what I actually wanted to change here.